src/RedisGraph/Node.php

```diff
@@ -104,4 +113,9 @@ private function getPropValueWithDataType($propValue)
         }
         return (int) $propValue;
     }
+
+    public function getQueryPredicate(): string
```

Comment:
this bit is important that i set it to default to MATCH to preserve existing behavior